### PR TITLE
Add new collective card design on discover page

### PR DESCRIPTION
**Pull request by JoyLubega**
resolve: opencollective/opencollective/issues/2455

screenshot:

Screen Shot 2020-01-03 at 6 11 06 PM

### CODE REVIEW

File: components/NewCollectiveCards.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+          }
+
+          .labels {
+            font-size: 12px;
+            text-align: left;
+            font-weight: 300;
+            color: #303233;
+            text-transform: capitalize;
+          }
+        `}
```
Comment (Betree) [inline components/NewCollectiveCards.js]: We prefer to use styled-components to style new components

**Comment by Betree** [comment]
@JoyLubega let me know when you're done, I'll make some final adjustments before merging

**Comment by JoyLubega** [comment]
@Betree  I am just adding cypress tests then I will be done. I started on them yesterday. I will be done today. I will ping you.

**Comment by JoyLubega** [comment]
@Betree  Am done

**Comment by Betree** [comment]
Thanks for adding a test :slightly_smiling_face:!

**Comment by JoyLubega** [comment]
@Betree  is it okay, if I squash the commits now

**Comment by Betree** [comment]
@JoyLubega It's fine, I'll squash them when merging

**Comment by znarf** [comment]
@Betree it was already the case before, so it's not a regression, but why is that page not rendering on the server side?

**Comment by Betree** [comment]
@znarf I'm not sure, maybe the fact that it's exported as a functional component?

**Pull request merged**